src/PrestaShopBundle/Resources/views/Admin/Module/Includes/modal_read_more_content.html.twig

```diff
@@ -118,6 +118,25 @@
                 {{ 'No description found for this module :('|trans({}, 'Admin.Modules.Notification') }}
               {% endif %}
             </p>
+            {% if module.attributes.multistoreCompatibility is defined and module.attributes.multistoreCompatibility is not null %}
+              <div class="module-readmore-multistore-content">
+                <h3>{{ 'Multistore compatibility:'|trans({}, 'Admin.Modules.Feature') }}</h3>
+                {% if module.attributes.multistoreCompatibility is same as(constant('\\Module::MULTISTORE_COMPATIBILITY_YES')) %}
+                  {{ 'Yes: this module is compatible with the multistore feature. It can be either:'|trans({}, 'Admin.Modules.Feature') }}
```

Comment:
```suggestion
                  {{ 'This module is compatible with the multistore feature. It can be either:'|trans({}, 'Admin.Modules.Feature') }}
```